flying-saucer-pdf/src/main/java/org/xhtmlrenderer/pdf/ITextOutputDevice.java

```diff
@@ -925,18 +925,15 @@ private void writeBookmarks(RenderingContext c, Box root, PdfOutline parent, Lis
     }
 
     private void writeNamedDestinations(RenderingContext c) {
-        Map idMap = getSharedContext().getIdMap();
+        Map<String, Box> idMap = getSharedContext().getIdMap();
         if ((idMap != null) && (!idMap.isEmpty())) {
```

Comment:
If `isMap` is guaranteed to never be null, would that simplify this entire method?

Reply:
done